### PR TITLE
Checkstyle

### PR DESCRIPTION
**Pull request by norwood**

### CODE REVIEW

**Comment by norwood** [comment]
depends on https://github.com/confluentinc/common/pull/47

File: core/src/main/java/io/confluent/rest/Application.java [hunk 1]
```diff
@@ -150,7 +158,7 @@ public Server createServer() throws RestConfigException {
 
     // Configure the servlet container
     ServletContainer servletContainer = new ServletContainer(resourceConfig);
-    FilterHolder servletHolder = new FilterHolder(servletContainer);
+    final FilterHolder servletHolder = new FilterHolder(servletContainer);
```
Comment (ewencp) [inline core/src/main/java/io/confluent/rest/Application.java]: Does this really complain about the `FilterHolder` but not the `ServletContainer` right above it? Or just unrelated cleanup?

Comment (norwood) [inline core/src/main/java/io/confluent/rest/Application.java]: it complains about this since `servletHolder` is not used near its declaration

**Comment by norwood** [comment]
retest this please
